packages/example/package.json

```diff
@@ -6,12 +6,11 @@
     "clean": "rm -rf ./dist/esm/* ./dist/types/*",
     "build": "../../node_modules/typescript/bin/tsc --project tsconfig.json --module ES2015 --outDir ./dist/esm --declaration --declarationDir ./dist/types",
     "add-person": "node dist/esm/add-person.js addressbook.bin",
-    "list-people": "node dist/esm/list-people.js addressbook.bin"
+    "list-people": "node dist/esm/list-people.js addressbook.bin",
+    "buf:generate": "buf generate"
   },
   "type": "module",
   "dependencies": {
-    "@bufbuild/protobuf": "0.1.1",
-    "@bufbuild/protoc-gen-es": "0.1.1",
```

Comment:
Ah yeah.  This is just temporary in my local setup I think because I have the libraries linked so I can run against my local changes.  I will revert this before it lands.